src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,24 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int armLength;
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator` class. Consider using private access modifier to encapsulate the `armLength` field.